include/pisa/topk_queue.hpp

```diff
@@ -76,6 +83,7 @@ struct topk_queue {
 
   private:
     float m_threshold;
+    float m_predicted_threshold;
```

Comment:
Instead of `m_predicted_threshold` can we call it `m_initial_threshold`?
It might not be "predicted" and I guess it can generate some confusion. The name `initial` abstracts out the way it was obtained, instead it indicates the heap is initialized to a value.

Reply:
I think I like "initialized". I think also we should document these things in the code to make it clear.